### PR TITLE
Fix Affine-Cipher encrypt and drypt func

### PR DESCRIPTION
**Pull request by shidian00**
[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### Describe your change:
- encrypt func should return alphabet instead of numbers string
- add checking for coprime as if coefficient 'a' is not coprime to 26 will reusultant in invalid result in inverse mod
- modify some test case 

- [ ] Add an algorithm?
- [x] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### Checklist:

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All new JavaScript files are placed inside an existing directory.
- [x] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
- [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: Ciphers/AffineCipher.js [hunk 1]
```diff
@@ -28,6 +28,27 @@ function inverseMod (a, m) {
   }
 }
 
+/**
+ * GetEuclidGCD Euclidean algorithm to determine the GCD of two numbers
```
Comment (appgurueu) [inline Ciphers/AffineCipher.js:32]: Please don't copy this but rather import the GCD func.

File: Ciphers/AffineCipher.js [hunk 2]
```diff
@@ -44,8 +65,13 @@ function isCorrectFormat (str, a, b) {
     throw new TypeError('Argument str should be String')
   }
 
+  if (!(GetEuclidGCD(a, 26) === 1)) {
+    throw new Error(a + ' is not coprime of 26 !!!')
```
Comment (appgurueu) [inline Ciphers/AffineCipher.js:69]: You don't need the exclamation marks.

Comment (shidian00) [reply Ciphers/AffineCipher.js:49]: i replace it with the import coprimecheck

File: Ciphers/AffineCipher.js (hunk 1 above)
Comment (shidian00) [reply Ciphers/AffineCipher.js]: Sorry initally when i import the js file, when debug it keep giving me error module not found, but during test it pass i not sure whether it correct or not...

File: Ciphers/AffineCipher.js (hunk 2 above)
Comment (appgurueu) [reply Ciphers/AffineCipher.js:49]: I was referring to the three exclamation marks in the error message.

**Review by appgurueu — APPROVED** [review]

**Pull request merged**